app/components/Header.tsx

```diff
@@ -100,9 +101,43 @@ export function Header() {
               >
                 Circulars
               </NavLink>,
-              <NavLink className="usa-nav__link" to="/docs" key="/docs">
-                Documentation
-              </NavLink>,
+              useFeature('SCHEMA') ? (
+                <>
+                  <NavDropDownButton
+                    to="/docs"
+                    type="button"
+                    key="docs"
+                    label="Documentation"
+                    isOpen={docsMenuIsOpen}
+                    onToggle={() => setDocsMenuIsOpen(!docsMenuIsOpen)}
+                    menuId="notices"
+                  />
+                  <Menu
+                    id="docs"
+                    isOpen={docsMenuIsOpen}
+                    items={[
+                      <Link
+                        key="docs"
+                        to="/docs"
+                        onClick={() => setDocsMenuIsOpen(!docsMenuIsOpen)}
+                      >
+                        Documentation
+                      </Link>,
```

Comment:
So there the nav drop down item Documentation has a menu item called Documentation? That doesn't make sense.

Reply:
Do you have a better name in mind? I just copied the original button into there. /docs brings the user to the About page, so  maybe About?

Reply:
No... maybe just don't make it a dropdown for now.